arnold/api/api_v1/endpoints/case.py

```diff
@@ -0,0 +1,49 @@
+from fastapi import APIRouter, Body, Depends, status
+from fastapi.responses import JSONResponse
+
+from arnold.crud import create, read
+from arnold.adapter.plugin import ArnoldAdapter
+from arnold.settings import get_arnold_adapter
+from arnold.models.database.case import Case
+
+router = APIRouter()
+
+
+@router.post(
+    "/case/",
+    response_description="Create a new case document",
+    status_code=status.HTTP_201_CREATED,
+    response_model=Case,
+)
+def create_case(
+    case: Case = Body(...),
+    adapter: ArnoldAdapter = Depends(get_arnold_adapter),
+) -> JSONResponse:
+    """Create a case document in the database."""
+    if read.case.get_case(case_id=case.id, adapter=adapter):
+        return JSONResponse(
+            status_code=status.HTTP_405_METHOD_NOT_ALLOWED,
```

Comment:
409 might be more descriptive

Reply:
I just learned about the HTTP_418_I_AM_A_TEAPOT... I think we should use this everywhere. :)